### ISSUE TITLE
Add navbar to Best Seller's page.

### ISSUE THREAD

**Issue by saloni-15** [saloni-15/Heel2Toe#81]
### Issue Description

- Include the same navbar as on the women's page and men's page on the best seller's page.
  - Add links to `HOME  MEN  WOMEN  NEW ARRIVALS  TRENDING  CONTACT`
- Use `best.css` for styling the page.
- Remove this top section on the `Best Seller's` page
   
![image](https://user-images.githubusercontent.com/75828760/135760682-56b0e97e-8ca8-477c-9350-40722e230b1f.png)

- Please include a screenshot/gif while creating the PR.

### Are you a Hacktoberfest'21 contributor

- [X] Yes, I'am Hacktoberfest'21 contributor

**Comment by bindu-r02** [comment]
@saloni-15 - May I work on this

**Comment by saloni-15** [comment]
sure @bindu-r02

**Comment by bindu-r02** [comment]
@saloni-15 - Is this fine?

![image](https://user-images.githubusercontent.com/72297585/135824943-a8e764e2-a671-409f-8797-78a7b97e2a19.png)

**Comment by saloni-15** [comment]
> @saloni-15 - Is this fine?
> 
> ![image](https://user-images.githubusercontent.com/72297585/135824943-a8e764e2-a671-409f-8797-78a7b97e2a19.png)

Yes the navbar looks good, but don't change the styling of carousel.
